src/actions/view.js

```diff
@@ -35,6 +36,16 @@ export const setEditModelField = (keys, value, storeValue, entityId, sectionName
 	payload: { keys, value, storeValue, entityId, sectionName, moduleName },
 });
 
+export const setFullEntityEditModel = (entityFullModel, moduleName) => ({
+	type: VIEW_SET_FULL_ENTITY_EDIT_MODEL,
+	payload: { entityFullModel, moduleName },
+});
+
+export const removeEditAllModelFields = (keys, entityId, sectionName, moduleName) => ({
+	type: VIEW_REMOVE_EDIT_ALL_MODEL_FIELDS,
+	payload: { keys, entityId, sectionName, moduleName },
```

Comment:
I believe we dont need that and mix of existing actions + dynamic edit state should allow you to work with any edit state